ports/esp32/mpconfigport.h

```diff
@@ -191,11 +191,11 @@ extern const struct _mp_obj_module_t uos_module;
 extern const struct _mp_obj_module_t mp_module_usocket;
 extern const struct _mp_obj_module_t mp_module_machine;
 extern const struct _mp_obj_module_t mp_module_network;
-extern const struct _mp_obj_module_t mp_module_bluetooth;
+extern const struct _mp_obj_module_t mp_module_ubluetooth;
 extern const struct _mp_obj_module_t mp_module_onewire;
 
 #if MICROPY_PY_BLUETOOTH
-#define BLUETOOTH_BUILTIN_MODULE              { MP_ROM_QSTR(MP_QSTR_bluetooth), MP_ROM_PTR(&mp_module_bluetooth) },
+#define BLUETOOTH_BUILTIN_MODULE              { MP_ROM_QSTR(MP_QSTR_ubluetooth), MP_ROM_PTR(&mp_module_ubluetooth) },
 #else
 #define BLUETOOTH_BUILTIN_MODULE
 #endif
```

Comment:
Maybe this stuff should be moved to `py/objmodule.c`, next to all the other extmod modules that are included if their config value is enabled?

Reply:
Seems reasonable. Done!